lua/lazyvim/plugins/ui.lua

```diff
@@ -198,7 +198,10 @@ return {
       }
 
       -- do not add trouble symbols if aerial is enabled
-      if vim.g.trouble_lualine and LazyVim.has("trouble.nvim") then
+      -- And allow it to be overriden for some buffer types (see autocmds)
+      local trouble_lualine_enabled = vim.b.trouble_lualine == nil or vim.b.trouble_lualine
+      trouble_lualine_enabled = trouble_lualine_enabled and vim.g.trouble_lualine
+      if trouble_lualine_enabled and LazyVim.has("trouble.nvim") then
```

Comment:
Can you change this to `if vim.g.trouble_luline_enabed ~= false and vim.b.trouble_lualine_enabled ~= false and LAzyVim.has....`

Reply:
Done. Thanks for the feedback!